status.json

```diff
@@ -5057,5 +5057,21 @@
     "ieStatus": {
       "text": "Under Consideration"
     }
+  },
+  {
+    "name": "Picture-in-Picture",
+    "category": "Multimedia",
+    "link": "https://wicg.github.io/picture-in-picture/",
+    "summary": "An API for allowing websites to create a floating video window always on top of other windows so that users may continue consuming media while they interact with other content sites, or applications on their device.",
+    "standardStatus": "Editor's draft",
+    "ieStatus": {
+      "text": "Under Consideration"
```

Comment:
@beaufortfrancois Did someone from Microsoft specified this?

Reply:
Yep. Feel free to send me an email at fbeaufort@google.com to know more.

Reply:
Done.